webpage/_data/navigation.yml

```diff
@@ -13,3 +13,9 @@ site_nav:
     children:
       - title: "Installing the IOOS conda environment"
         url: /other_resources
+      - title: "Australian ocean data network portal user guide"
```

Comment:
Should I use `AODN` instead of Australian ocean data network to make this shorter?